app/_locales/en/messages.json

```diff
@@ -5320,6 +5320,9 @@
   "stake": {
     "message": "Stake"
   },
+  "stateCorruptionDetectedErrorMessage": {
+    "message": "MetaMask's state file is damaged or missing. The files are possibly recoverable, so don't uninstall the MetaMask extension! You should contact MetaMask support."
```

Comment:
Let's add a support link here. It'll make things easier for our users if they need help.

Example:
https://github.com/MetaMask/metamask-extension/blob/fcace764382856e4a5369898d5a32386fe4cba98/app/_locales/en/messages.json#L614-L617

Reply:
A support link is added with a separate string, see : https://github.com/MetaMask/metamask-extension/pull/30684/files#diff-7f06df2eee37f673db40d35818304c95e0339b1d5b049ea3ea831064f1a2ac68R90-R99